CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+1.3.6
```

Comment:
I think this deserves to be a minor release at least (1.4.0)

Reply:
done